client/homebrew/pages/newPage/newPage.jsx

```diff
@@ -84,6 +84,9 @@ const NewPage = createClass({
 		if(brew.style)
 			localStorage.setItem(STYLEKEY, brew.style);
 		localStorage.setItem(METAKEY, JSON.stringify({ 'renderer': brew.renderer, 'theme': brew.theme, 'lang': brew.lang }));
+		if(window.location.pathname != '/new') {
+			window.location.pathname = '/new';
```

Comment:
This appears to trigger a page reload. `window.history.replaceState();` may be more appropriate here.

Reply:
Didn't spot that with local. Too fast I guess. :)